modules/runtime/nuxeo-launcher-commons/src/main/java/org/nuxeo/launcher/config/ServerConfigurator.java

```diff
@@ -111,16 +111,14 @@ public ServerConfigurator(ConfigurationGenerator configurationGenerator) {
      * @return true if server configuration files already exist
      */
     protected boolean isConfigured() {
-        return new File(generator.getNuxeoHome(), getTomcatConfig()).exists();
-    }
-
-    /**
-     * @since 5.4.2
-     * @return Path to Tomcat configuration of Nuxeo context
-     */
-    public String getTomcatConfig() {
-        return "conf" + File.separator + "Catalina" + File.separator + "localhost" + File.separator + getContextName()
-                + ".xml";
+        return generator.getNuxeoHome()
+                        .toPath()
+                        .resolve("conf")
+                        .resolve("Catalina")
+                        .resolve("localhost")
+                        .resolve(getContextName() + ".xml")
+                        .toFile()
+                        .exists();
```

Comment:
Maybe `Files.exists(generator.getNuxeoHome().toPath().resolve(Path.of("conf", "Catalina", "localhost", getContextName() + ".xml")))`